a5/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # rosindustrial/a5:kinetic
-FROM rosindustrial/noether-webtools:kinetic
+FROM rosindustrial/noether:kinetic
 MAINTAINER Austin.Deric@gmail.com
 RUN apt-get update && apt-get install -y \
-  ros-kinetic-stage \
-  sudo
+  ros-kinetic-stage
+RUN apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
If I may suggest: make this `RUN` (the `apt-get clean ..`) part of the one before it. So:

```Docker
RUN apt-get update && apt-get install -y \
  ros-kinetic-stage \
  && \
  apt-get clean && \
  rm -rf /var/lib/apt/lists/*
```

that way you avoid having any intermediary layers in which the apt cache and packages are still around.